Assets/Script/GameManager.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+using System;
+
+public class GameManager : MonoBehaviour
+{
+    public Text PlayTime;
+    private TimeSpan startTime = new TimeSpan(0, 10, 00);
+
+    private void Awake()
+    {
+
+    }
+
+    private void Start()
+    {
+        InvokeRepeating("CountDown", 0, 1);
+    }
+
+
+
+    private void CountDown()
+    {
+        startTime = startTime.Subtract(new TimeSpan(0, 0, 1));
```

Comment:
```suggestion
        remainingTime = startTime - TimeSpan.FromSeconds(1);
```

- `-` operator overloading: difference between java and C# (include C++)
- Create TimeSpan object from static method `FromSeconds` => more intuitive
- startTime must be static value, replaced to remaining time

Reply:
안그래도 차후에 남은 시간으로 이벤트 처리를 해야했었는데 사전에 콕 집어주셨군요 ㅎㅎ

Reply:
그리고 메모리 영역을 바꿔줌으로써 좀더 안정화를 높일 수 있는 것도 좋은 팁인 것 같습니다!